perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -293,9 +326,145 @@ def _MakeSwiftCommandPrefix(auth_url, tenant_name, username, password):
   return ' '.join(options)
 
 
+def _ParseMultiStreamResults(raw_result, metadata=None):
+  """Read results from the api_multistream worker process.
+
+  Args:
+    raw_result: string. The stdout of the worker process.
+    metadata: dict. Base sample metadata
+
+  Returns:
+    A list of sample.Sample objects.
+  """
+
+  if metadata is None:
+    metadata = {}
+
+  records = json.loads(raw_result)
+  results = []
+
+  for record in records:
+    metric = 'Multi-stream %s KB %s' % (record['size'], record['operation'])
```

Comment:
Metric names are changed to "Multi-stream {upload|download} latency".

Yes, I can add throughput metrics in the next PR, but I'd like to think more about the design. I have tried very deliberately to store the raw unprocessed data as samples, so I will have as much flexibility as possible in the analysis step. I think it makes sense to also publish processed values as samples, but it's worth talking about.